src/o365/spo/commands/web/web-set.ts

```diff
@@ -142,6 +147,10 @@ class SpoWebSetCommand extends SpoCommand {
         option: '--megaMenuEnabled [megaMenuEnabled]',
         description: 'Set to \'true\' to change the menu style to megamenu. Set to \'false\' to use the cascading menu style',
         autocomplete: ['true', 'false']
+      },
+      {
+        option: '--footerEnabled [footerEnabled]',
```

Comment:
Given that this option accepts only `true` and `false`, we could add autocomplete to simplify specifying the correct value.